{{cookiecutter.role_name}}/requirements.txt

```diff
@@ -1,3 +1,6 @@
 ansible>=2.9.0,<2.10.0
-molecule==2.22
+ansible-lint
+flake8
+molecule>=3.0.2
 docker
+yamllint
```

Comment:
It looks like right now testinfra is also required in requirements.txt due to this bug:
https://github.com/ansible-community/molecule/issues/2597